tests/test_object_group.rb

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2017 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/object_group'
+require_relative '../lib/cisco_node_utils/object_group_entry'
+
+# TestObject_group - Minitest for Object_group
+# node utility class
+class TestObjectGroup < CiscoTestCase
+  def setup
+    super
+    @og_name_v4 = 'test-my-v4'
+    @og_name_v6 = 'test-my-v6'
+    @og_name_port = 'test-my-port'
+    no_og_list_my
+  end
+
+  def teardown
+    no_og_list_my
+    super
+  end
+
+  def no_og_list_my
```

Comment:
I think it would be simpler to get the objects and call the destroy method here:

For example: https://github.com/cisco/cisco-network-node-utils/blob/develop/tests/test_ace.rb#L35